src/tokens/build.mjs

```diff
@@ -27,7 +27,7 @@ const getStyleDictionaryConfig = (theme) => {
     ],
     platforms: {
       css: {
-        buildPath: `tokens/${lowerCasedTheme}/`,
+        buildPath: `src/tokens/${lowerCasedTheme}/`,
```

Comment:
@kurtdoherty The Export path also required an update to the export token (CSS/ts) path.